plugins/obs-libfdk/CMakeLists.txt

```diff
@@ -13,9 +14,6 @@ add_library(obs-libfdk MODULE)
 add_library(OBS::libfdk ALIAS obs-libfdk)
 
 target_sources(obs-libfdk PRIVATE obs-libfdk.c)
+target_link_libraries(obs-libfdk PRIVATE OBS::libobs Libfdk::Libfdk)
```

Comment:
Add a newline before? You typically space these out even if they're single lines.